p2p/libp2p/netMessenger.go

```diff
@@ -145,6 +145,7 @@ func NewNetworkMessenger(args ArgsNetworkMessenger) (*networkMessenger, error) {
 	ctx, cancelFunc := context.WithCancel(context.Background())
 	h, err := libp2p.New(ctx, opts...)
 	if err != nil {
+		cancelFunc()
```

Comment:
:+1: 
This was fixed in feature branch that will be merged in the near future:
https://github.com/ElrondNetwork/elrond-go/blob/0b35556653d474d3caf03eb37bb372b51e47f378/p2p/libp2p/netMessenger.go#L150

Reply:
ok, I'll close it.

Reply:
Let's merge this as it resolves a linter issue.